Schema.sql

```diff
@@ -0,0 +1,65 @@
+DROP DATABASE IF EXISTS `wavegod`;
+SET default_storage_engine=InnoDB;
+SET NAMES utf8mb4 COLLATE utf8mb4_unicode_ci;
+
+CREATE DATABASE IF NOT EXISTS wavegod
+    DEFAULT CHARACTER SET utf8mb4
+    DEFAULT COLLATE utf8mb4_unicode_ci;
+USE wavegod;
+
+create Table `Picture`(
+    photoID int NOT NULL AUTO_INCREMENT,
+    photo varchar(40),
+    dateOfCreation datetime,
+    PRIMARY KEY (photoID)
+);
+
+create table `Fan` (
+    fanID int NOT NULL AUTO_INCREMENT,
+    firstname varchar(40) NOT NULL,
+    surname varchar(40) NOT NULL,
+    username varchar(40) NOT NULL,
+    password varchar (40) NOT NULL,
+    email varchar(40) NOT NULL,
+    street varchar(100) NOT NULL,
+    city varchar(40) NOT NULL,
+    country varchar(40) NOT NULL,
+    telephoneNo int NOT NULL,
+    dateOfBirth date NOT NULL,
+    dateOfCreation datetime NOT NULL,
+    photoID int NOT NULL,
+    PRIMARY KEY (fanID),
+    FOREIGN KEY (photoID) REFERENCES Picture(photoID)
+);
+
+create Table `Album`(
```

Comment:
Please use UPPERCASE for all SQL specific key words such as CREATE, TABLE, FROM, SELECT etc